### ISSUE TITLE
Obsolete AngularJS and JQuery scripts

### ISSUE THREAD

**Issue by nmg196** [RickStrahl/Westwind.Globalization#197]
Hi,
One of our sites which uses WestWind Globalization failed an annual pen test because it uses old versions of AngularJS and JQuery which have known security vulnerabilities.   Has anyone attempted to upgrade AngularJS or JQuery to the current version? 

Is this project still being maintained? There seemt be a lot of open issues and PRs and .NET Core 3.1 seems unsupported by default.

Thanks
Nick

**Comment by lvmajor** [comment]
@nmg196 yeah there seems to not be much activity by the maintainer, but a "pre-release" was made on Nuget which adds support for .net core 3+ (see on [Nuget](https://www.nuget.org/packages/Westwind.Globalization/3.1.0-preview1))

**Comment by nmg196** [comment]
We have now ported Angular to 1.8 but not submitted a PR yet as I doubt it will get merged. Let me know if you need it.
